targets/TARGET_NUVOTON/TARGET_M480/trng_api.c

```diff
@@ -77,21 +82,21 @@ void trng_free(trng_t *obj)
 int trng_get_bytes(trng_t *obj, uint8_t *output, size_t length, size_t *output_length)
 {
     (void)obj;
-
-    *output_length = 0;
-    if (length < 32) {
-        unsigned char tmpBuff[32];
+    unsigned char tmpBuff[32];
+    size_t cur_length = 0;
+    
+    for (unsigned i = 0; i < (length/32); i++) {
```

Comment:
Consider using `sizeof(tmpBuff)` instead of `32`.

Reply:
Consider also using spaces around binary operators like `/`, for consistency with the style of other code in this file.

Reply:
@Patater Update the code in new commit d8a9e35